memory/src/main/scala/filodb.memory/format/vectors/Histogram.scala

```diff
@@ -480,6 +518,148 @@ final case class GeometricBuckets(firstBucket: Double,
   }
 }
 
+object Base2ExpHistogramBuckets {
+  // TODO: make maxBuckets default configurable; not straightforward to get handle to global config from here
+  val maxBuckets = 200
```

Comment:
object HistogramBuckets {
  val MAX_BUCKETS = 8192
}


Should it match with above ?

Reply:
No - I don't want to be that liberal yet :) . The intent was to match Otel JDK maxBuckets default. We can expand later if there is an explicit ask. Harder to close doors and restrict than to open it up slowly.